tools/cd_scripts/create_binaries.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The script is taking around 20-25 minutes on n2-standard-32 machine.
+gcloud compute instances describe create-gcsfuse-binaries --zone us-west1-b --flatten="metadata[RELEASE_VERSION]" >>  release_version.txt
```

Comment:
we are writing release version to bucket after building gcsfuse package. Do you think we can move that stage prior to running the script and use gsutil command instead of passing extra metadata here?

Reply:
Than should we pass bucket name in details.txt to as we need bucket name also?

Reply:
I think metadata is fine only as if in future if we add more details in details.txt may get inconsistent. What do you think @sethiay ?

Reply:
I believe in that case we should change other scripts to use metadata as well to maintain consistency. Let's confirm with @sethiay on this before merging.

Everything else LGTM.

Reply:
```
 believe in that case we should change other scripts to use metadata as well to maintain consistency. Let's confirm with @sethiay on this before merging.
```
Yeah if so require, I'll raise separate bug for that as it is not related to arm64 change.